### PR TITLE
fix: begin epoch 3.0 block times at start height - 1

### PR DESCRIPTION
**Pull request by obycode**
This allows the extra time needed during the switchover to Nakamoto.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ services:
 
           SLEEP_DURATION=$${MINE_INTERVAL}
           BLOCK_HEIGHT=$$(bitcoin-cli -rpcconnect=bitcoind getblockcount)
-          if [ "$${BLOCK_HEIGHT}" -ge "$${STACKS_30_HEIGHT}" ]; then
+          if [ "${BLOCK_HEIGHT}" -ge $(( ${STACKS_30_HEIGHT} - 1 )) ]; then
```
Comment (zone117x) [inline docker-compose.yml:103]: Does this work for you locally? I think `$` has to be doubled (to escape) in this yaml context?

Comment (obycode) [reply docker-compose.yml:103]: You're right. I copy/pasted this from the k8s repo and forgot to make that change. Will push the fix shortly.

Comment (obycode) [reply docker-compose.yml]: Fixed in 3c7b5a720c4fc2c42bd56737b22cbbcc5f48fe37

**Review by janniks — APPROVED** [review]
Fine for me 👍 Epoch 3 times will typically have slower times for bitcoin

**Review by zone117x — APPROVED** [review]

**Comment by janniks** [comment]
@obycode what behavior where you seeing in these cases? Would the network/node stall at the Epoch3 switch-height, or later?

**Comment by obycode** [comment]
> what behavior where you seeing in these cases? Would the network/node stall at the Epoch3 switch-height, or later?

Just a missed block at the switch height, which is not a problem, but could cause unnecessary confusion.

**Pull request merged**